services/brig/brig.cabal

```diff
@@ -1,10 +1,10 @@
-cabal-version: 2.0
+cabal-version: 1.12
```

Comment:
Why does that happen? Your hpack version seems to be the same, so why does it generate the file for a different Cabal version?

Reply:
hpack figures out that without internal libraries, this cabal file will even work with older `cabal-version`, so it updates that.